frontend/packages/dev-console/src/components/topology/bindable-services/topology-data-model.ts

```diff
@@ -0,0 +1,55 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { getRhoasServiceBindingEdges } from '@console/rhoas-plugin/src/topology/rhoas-data-transformer';
+import { getTopologyNodeItem } from '@console/topology/src/data-transforms/transform-utils';
+import { TopologyDataObject, TopologyDataResources } from '@console/topology/src/topology-types';
+import { TYPE_BINDABLE_NODE } from '../const';
+import { getBindableServicesList } from './fetch-bindable-services-utils';
+
+const getTopologyBindableServiceNodes = (resources: K8sResourceKind[]) => {
+  const nodes = resources.map((obj) => {
+    const data: TopologyDataObject = {
+      id: obj.metadata.uid,
+      name: obj.metadata.name,
+      type: TYPE_BINDABLE_NODE,
+      resource: obj,
+      resources: { obj },
```

Comment:
These utilities are duplicates from the data-transformer file https://github.com/openshift/console/pull/9713/files#diff-49655a575ab5cacc79a9c672170b0ef44fe3cd3628dbbee983cb5e5dd80da1f1

Reply:
Had forgotten to remove : p